src/components/introduction.js

```diff
@@ -1,32 +0,0 @@
-import React from 'react';
```

Comment:
i changed the suffix from jsx to js so that it'd get caught by eslint in https://github.com/CMSgov/qpp-submissions-docs/pull/55 - jsx syntax in this file still gets processed correctly with a .js suffix.